iron-iconset-svg.html

```diff
@@ -69,6 +69,10 @@
       }
 
     },
+    
+    attached: function() {
+      this.style.display = 'none';
```

Comment:
word, I wonder if using `created ` instead of attached then might be better, just to shove this as early as possible in the life cycle of the elm, to prevent crazy weird style/height flicker